addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3465,19 +3465,18 @@ var SnippetsMenu = Widget.extend({
     _checkEditorToolbarVisibility: function (e) {
         const $toolbarContainer = this.$('#o_we_editor_toolbar_container');
         const $toolbarTableContainer = this.$('#o-we-editor-table-container');
-        const docSelection = document.getSelection();
-        const $currentSelectionTarget = docSelection && docSelection.rangeCount > 0 ? $(docSelection.getRangeAt(0).commonAncestorContainer) : $();
-        // Do not  toggle visibility if the target is inside the toolbar ( eg. during link edition).
+        const selection = this.options.wysiwyg.odooEditor.document.getSelection();
+        const range = selection && selection.rangeCount && selection.getRangeAt(0);
+        const $currentSelectionTarget = range ? $(range.commonAncestorContainer) : $();
```

Comment:
I know it was already like this but I think now might be a good time to change it to `$(range && range.comonAncestorContainer)`.